src/mymonero/monero_utils/monero_sendingFunds_utils.js

```diff
@@ -28,10 +28,10 @@
 //
 "use strict";
 //
-const monero_config = require("./monero_config");
-const monero_amount_format_utils = require("./monero_amount_format_utils");
+const monero_config = require("@mymonero/mymonero-money-format/monero_config");
+const monero_amount_format_utils = require("@mymonero/mymonero-money-format");
 const monero_paymentID_utils = require("./monero_paymentID_utils");
```

Comment:
Same question as before but for the other libs.

Reply:
Same answer for the others. The libs we have copied all have `async` added to them.